vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS-v2.1.ttl

```diff
@@ -2546,6 +2546,23 @@ qkdv:A0E2L0I0M-1H0T4D0
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimensionvector> ;
   rdfs:label "A0E2L0I0M-1H0T4D0" ;
 .
+qkdv:A0E2L0I0M0H0T1D0
```

Comment:
No problem with this dimension vector, but it has nothing to do with Voltage or Voltage Loss as far as I can tell.

Reply:
Then lets fix the A2-HR reference and quantity kind.